Documentation/guides/messages/adb0000.md

```diff
@@ -1,20 +1,24 @@
-# Compiler Error/Warning ADB0000
+---
+title: Compiler error/warning ADB0000
+ms.date: 07/04/2018
+---
+# Compiler error/warning ADB0000
 
 This message indicates that `adb` (Android Debug Bridge) reported an
-unhandled/unexpected error or warning. `adb` is part of the Android
-SDK and is used internally by Xamarin.Android to communicate with
-Android emulators and devices. Learn more about `adb` from the
-[Android documentation][adb].
+unhandled/unexpected error or warning.  `adb` is part of the Android SDK and is
```

Comment:
Do we need to "re-format" *everything*?  The only changes here are "whitespace" changes -- wider line lengths, two spaces after a `.`, etc.  No "actual" content has changed.  (Some minor formatting changes in subsequent paragraphs.)

Is it necessary to do all this?

Was this reformatting done manually or via tooling?

Reply:
This reformatting was done manually, but I didn't have a strong opinion that motivated it, so in the next draft I'll plan to go back to using the original spacing after `.` for each document and only rewrap a paragraph if the line length has changed.